mutiny-core/src/test_utils.rs

```diff
@@ -10,19 +10,25 @@ macro_rules! log {
 pub(crate) use log;
 use rexie::Rexie;
 
-use crate::gossip::GOSSIP_DATABASE_NAME;
 use crate::indexed_db::MutinyStorage;
-
-pub fn cleanup_test() {
-    LocalStorage::clear();
-}
+use crate::{gossip::GOSSIP_DATABASE_NAME, logging};
 
 pub async fn cleanup_gossip_test() {
-    cleanup_test();
     Rexie::delete(GOSSIP_DATABASE_NAME).await.unwrap();
 }
 
 pub async fn cleanup_wallet_test() {
-    cleanup_test();
     MutinyStorage::clear().await.unwrap();
 }
+
+pub async fn cleanup_logging_test() {
+    logging::clear().await.unwrap();
+}
```

Comment:
should make these private so we make sure we always call `cleanup_all` just to be safe